pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -52,8 +52,8 @@ func (r *resourceAllocationScorer) score(
 	if r.resourceToWeightMap == nil {
 		return 0, framework.NewStatus(framework.Error, "resources not found")
 	}
-	requested := make(resourceToValueMap, len(r.resourceToWeightMap))
-	allocatable := make(resourceToValueMap, len(r.resourceToWeightMap))
+	requested := make(resourceToValueMap, len(resourceToValueMap))
+	allocatable := make(resourceToValueMap, len(resourceToValueMap))
```

Comment:
`resourceToValueMap` is a local type, it's illegal to use `len(resourceToValueMap)`.